bundles/org.openhab.automation.jrubyscripting/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+	<config-description uri="automation:jruby">
+
+		<parameter-group name="system">
+			<label>System Properties</label>
+			<description>This group defines JRuby system properties.</description>
+			<advanced>true</advanced>
+		</parameter-group>
+
+		<parameter-group name="environment">
+			<label>Ruby Environment</label>
+			<description>This group defines Ruby's environment.</description>
+			<advanced>false</advanced>
+		</parameter-group>
+
+		<parameter-group name="gems">
+			<label>Gems</label>
+			<description>This group defines the list of GEMS to install.</description>
+			<advanced>false</advanced>
+		</parameter-group>
+
+		<parameter name="local_context" type="text" required="false" groupName="system">
+			<label>Context Instance Type</label>
+			<description>The local context holds Ruby runtime, name-value pairs for sharing variables between Java and Ruby. See
+				https://github.com/jruby/jruby/wiki/RedBridge#Context_Instance_Type for options and details.</description>
+			<default>singlethread</default>
+			<options>
+				<option value="singleton">Singleton</option>
+				<option value="threadsafe">ThreadSafe</option>
+				<option value="singlethread">SingleThread</option>
+				<option value="concurrent">Concurrent</option>
+			</options>
+		</parameter>
+
+		<parameter name="local_variable" type="text" required="false" groupName="system">
+			<label>Local Variable Behavior</label>
+			<description>Defines how variables are shared between Ruby and Java. See
+				https://github.com/jruby/jruby/wiki/RedBridge#local-variable-behavior-options for options and details.</description>
+			<default>transient</default>
+			<options>
+				<option value="transient">Transient</option>
+				<option value="persistent">Persistent</option>
+				<option value="global">Global</option>
+			</options>
+		</parameter>
+
+		<parameter name="gem_home" type="text" required="false" groupName="environment">
+			<label>GEM_HOME</label>
+			<description>Location ruby gems will be installed and loaded, directory will be created if missing and gem installs
+				are specified</description>
+			<default></default>
+		</parameter>
+
+		<parameter name="rubylib" type="text" required="false" groupName="environment">
+			<label>RUBYLIB</label>
+			<description>Search path for user libraries. Separate each path with a colon (semicolon in Windows).</description>
+		</parameter>
+
+		<parameter name="gems" type="text" required="false" groupName="gems">
+			<label>Gems</label>
+			<description>Comma separated list of Ruby Gems to install.</description>
```

Comment:
Can you make the capitalization of "gems" and "ruby" a bit more consistent?

There's now:

* gems, Gems, GEMS
* ruby, Ruby

After making it consistent, can you also add a default i18n properties file so these strings can be translated with Crowdin?

There is a handy Maven plugin PR that can help you with generating the file, see: https://github.com/openhab/openhab-core/pull/2544

The label for the service still needs to be manually added, see for instance [openhabcloud.properties](https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.io.openhabcloud/src/main/resources/OH-INF/i18n/openhabcloud.properties)